src/collection/collection.service.ts

```diff
@@ -677,7 +682,9 @@ export class CollectionService {
 
 			return collection.thumbnail_path;
 		} catch (err) {
-			console.error('Failed to get the thumbnail path for collection', err, { collection });
+			console.error('Failed to get the thumbnail path for collection', err, {
```

Comment:
console.error(new CustomError('Failed...

Zo hebben we altijd 1 plaats waar alle errors kunnen opgevangen worden, moesten we ze ooit willen loggen in een remote systeem